homeassistant/components/prometheus/__init__.py

```diff
@@ -318,6 +323,32 @@ def _handle_climate(self, state):
                     float(action == current_action)
                 )
 
+    def _handle_humidifier(self, state):
+        humidity = state.attributes.get(ATTR_HUMIDITY)
+        if humidity:
+            metric = self._metric(
+                "humidity", self.prometheus_cli.Gauge, "Target Relative Humidity"
```

Comment:
Could you please make metric name and description more expressive? Specifically:
1. Please prefix metric name with the domain (`humidifer_`).
2. Please specify whether this is current humidity or target humidity.
3. If this is relative humidity, I assume the values are in percents (e.g. 0 to 100)? If so, please indicate this as well.

As the result, metric name should be something like `humidifer_target_humidity_percent`.